app/workers/metrics/record_daily_usage_worker.rb

```diff
@@ -27,7 +27,7 @@ def perform
       DatadogStatsClient.count("articles.first_past_24h", first_articles_past_24h, tags: ["resource:articles"])
 
       # Users who signed up in the past 24 hours who have made at least 1 comment so far
-      new_users_min_1_comment_past_24h = User.where("comments_count >= ? AND created_at > ?", 1, 24.hours.ago).size
+      new_users_min_1_comment_past_24h = User.where("comments_count >= ? AND registered_at > ?", 1, 24.hours.ago).size
```

Comment:
Since we are going to start using this `registered_at` column right away we need to make sure we update it to match, I would think, `created_at` for all existing users.

Reply:
Yeah, my rationale for not including the data file was that it was just one more thing in a big PR and this functionality only gets triggered once per day. One less moving piece within this pull request.

How about this: After this gets merged, I whip up the data script as a separate PR?

Reply:
@mstruve I created #9331 to address this.

Let me know if I need to do anything differently... I actually haven't made a data script yet so I'm not sure I need to do anything else. (Docs are super great though 😄)

Reply:
Works for me!